pkg/bindings/kube/kube1.go

```diff
@@ -0,0 +1,97 @@
+package kube1
```

Comment:
The package should just be called `kube`.

The API `pkg/bindings/play` should probably call the functions here and the types should alias. This will reduce redundancy.